Include/Acidanthera/Protocol/OcBootEntry.h

```diff
@@ -0,0 +1,140 @@
+/** @file
+  OpenCore Boot Entry Protocol.
+
+  Copyright (C) 2021, Mike Beaton. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-3-Clause
+**/
+
+#ifndef OC_BOOT_ENTRY_PROTOCOL_H
+#define OC_BOOT_ENTRY_PROTOCOL_H
+
+#include <Uefi.h>
+#include <Library/OcBootManagementLib.h>
+
+#include <Protocol/SimpleFileSystem.h>
+
+/*
+  8604716E-ADD4-45B4-8495-08E36D497F4F
+*/
+#define OC_BOOT_ENTRY_PROTOCOL_GUID                                                 \
+  {                                                                                 \
+    0x8604716E, 0xADD4, 0x45B4, { 0x84, 0x95, 0x08, 0xE3, 0x6D, 0x49, 0x7F, 0x4F }  \
+  }
+
+/*
+  OC_BOOT_ENTRY_PROTOCOL revision.
+  Needs to be incremented when OC_PICKER_ENTRY structure is updated.
+*/
+#define OC_BOOT_ENTRY_PROTOCOL_REVISION 0
```

Comment:
Better start with 1.

Reply:
Yes, was kind of leaving it at 0 until ready to merge. But will probably forget, will up it now. ;-)